packages/website/src/pipeline/index.ts

```diff
@@ -6,11 +6,24 @@ import runWebsite from './stages/run-website';
 
 export default () => {
   const initialConfiguration = {};
-  return scanMetadata(initialConfiguration, {})
-    .then(result => generateWebsiteInfo(result, {}))
-    .then(result => generatePages(result, {}))
-    .then(result => buildWebsite(result, {}))
-    .then(result => runWebsite(result, {}))
+  return scanMetadata(initialConfiguration)
+    .then(result => generateWebsiteInfo(result))
+    .then(() =>
+      generatePages({
```

Comment:
It is the most useless non-functional config ever conceived. It does absolutely nothing.

... I could just comment it out, but it would mean that it would never get run in the single integration test that exists